ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -264,6 +275,34 @@ private void validateConfig(final JsonRpcConfiguration config) {
     return resultFuture;
   }
 
+  private Handler<HttpConnection> connectionHandler() {
+
+    return connection -> {
+      if (activeConnectionsCount.get() >= maxActiveConnections) {
+        LOG.warn(
+            "Max Active Connections limit {} reached. Rejecting all new connections.",
+            maxActiveConnections);
+        // disallow new connections to prevent DoS
+        LOG.warn(
+            "Rejecting new client connection: remoteAddress: {} connection count: {}",
+            connection.remoteAddress(),
+            activeConnectionsCount.incrementAndGet());
+        connection.close();
+      } else {
+        LOG.warn(
```

Comment:
This shouldn't be a warning. Change it to a debug message.

And we can change the msg slightly:
```
Opened HTTP connection from 192.160.1.1. Total of active HTTP connections: 12/80.
```